README.md

```diff
@@ -137,6 +137,24 @@ the `extra` section of the module's `composer.json` like this:
 }
 ````
 
+You can also provide system configuration for your module:
```

Comment:
This example may be confusing because it relates to the core phpList 4 module, rather than an optional or third part module (even though they are technically managed in the same way). Maybe an example of a hypothetical third party module would be better to make it appear more immediately relevant? Or configuration of the optional REST API module.

Reply:
@samtuke I've moved the section that explains that the key always needs to be `phplist/phplist4-core`. Does this help?